notfound/utils.py

```diff
@@ -1,6 +1,12 @@
 import docutils
 import re
-from sphinx.builders.html import DirectoryHTMLBuilder
+import sphinx
+
+# Sphinx <2 Compatibility
+if int(sphinx.__version__[0]) >= 2:
```

Comment:
Please, use `sphinx.version_info` here as we are already doing at https://github.com/readthedocs/sphinx-notfound-page/blob/master/notfound/extension.py#L222